user_guide_src/add-edit-this-page

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env php
+<?php declare(strict_types=1);
+
+if ($argc === 1) {
+    echo 'Usage: ' . $argv[0] . ' <html-dir>' . PHP_EOL;
+    echo '  e.g: ' . $argv[0] . ' user_guide_src/build/html/' . PHP_EOL;
+
+    exit(1);
+}
+
+$dir = realpath($argv[1]);
+
+if ($dir === false) {
```

Comment:
check also if `$dir` is really a directory

Reply:
Fixed.